From fcaea5fc4b3d7a06e273c147e62a7a003bdb59c7 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 May 2020 22:58:12 +0300
Subject: [PATCH] [SW-0] ppboot/preboot: fix copyright and license

Since we run checkpatch on this project, update app/ppboot and
app/preboot to comply with the new rules.

Change-Id: I02438b37e88f6626c46eb90541f07a5df670e13c
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ppboot.c b/app/ppboot.c
index 723c396..fd9edee 100644
--- a/app/ppboot.c
+++ b/app/ppboot.c
@@ -1,8 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0+
 /*
- * (C) Copyright 2017 HabanaLabs, Ltd.
- * Noam Camus <noamikong@gmail.com>
+ * Copyright (C) 2017-2020 HabanaLabs Ltd.
+ * All Rights Reserved.
  *
- * SPDX-License-Identifier:     GPL-2.0+
  */
 
 #include <psoc_global_conf_regs.h>
diff --git a/app/preboot.c b/app/preboot.c
index 93c6ab9..fc1666e 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -1,8 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0+
 /*
- * (C) Copyright 2017 HabanaLabs, Ltd.
- * Noam Camus <noamikong@gmail.com>
+ * (C) Copyright 2017-2020 HabanaLabs, Ltd.
+ * All rights reserved.
  *
- * SPDX-License-Identifier:     GPL-2.0+
  */
 
 #include "auto_generated.h"
